fecgov/fec-cms
# Replace topojson

## Summary (required)
- Resolves #5570 

This PR removes topojson and replaces it with topojson client. 

### Required reviewers - 2 developers 

## Impacted areas of the application

General components of the application that this PR will affect:

-  maps 


## How to test
1. checkout this branch 
2. `npm i && npm run build`
3. `cd fec/`
4. `./manage.py runserver`
5. Check various maps to ensure they still work:
     - http://127.0.0.1:8000/data/candidates/president/presidential-map/
     - http://127.0.0.1:8000/data/elections/?cycle=2022&state=TX&district=19
     - http://127.0.0.1:8000/data/candidate/S0GA00559/?cycle=2020&election_full=true&tab=raising 
               - under individual contributions, group by state
